zerver/tests/tornado_tests.py

```diff
@@ -0,0 +1,267 @@
+import time
+
+import ujson
+from django.http import HttpRequest, HttpResponse
+from tornado.gen import Return
+from tornado.httpclient import HTTPRequest
+
+from zerver.lib.event_queue import fetch_events
+from zerver.lib.test_helpers import POSTRequestMock
+from zerver.lib.test_classes import ZulipTestCase
+
+from zerver.models import UserProfile
+
+from tornado import gen
+from tornado.testing import AsyncHTTPTestCase, gen_test
+from tornado.web import Application
+from tornado.websocket import websocket_connect
+
+from zerver.management.commands.runtornado import create_application
+from zerver.tornadoviews import get_events_backend
+
+from six.moves.http_cookies import SimpleCookie
+
+from typing import Any, Callable, Generator, Optional
+
+
```

Comment:
Assuming we need to be using the Tornado unit testing system, can you add a docstring/comment at the top of this file just explaining that it's using the Tornado test system and thus one needs to use things like `@gen.test` to tag tests (with links to the docs)?  I think otherwise people will be confused as to why this file is so different from the other test files.